src/components/Hero/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Hero.scss';
+
+const Hero = props => (
+  <div className="hero-wrap">
```

Comment:
Note to reviewers: taken from https://github.com/edx/edx-mktg/blob/94523ea955e170ce462cf68a2c1637463fd51608/docroot/sites/all/themes/atedx/scripts/react/pages/masters_product_page/MastersProductPage.jsx#L545